### ISSUE TITLE
Deinstallation not working

### ISSUE THREAD

**Issue by Braintelligence** [splitbrain/dokuwiki-plugin-talkpage#5]
When trying to deinstall the discussion plugin it doesn't work. You see a red alert wrap but it doesn't contain any error message.

**Comment by splitbrain** [comment]
Just tried it. Installed and uninstalled just fine. Without further info I can't help

**Comment by Braintelligence** [comment]
I'll provide anything you need but I just don't know what info might be relevant. Please tell me what you need to know.

**Comment by Braintelligence** [comment]
One thing I just remembered is that I added translation files to the addon folder. Might that be the reason?

**Comment by splitbrain** [comment]
Yes. That might be the case. Check the permissions of those files.

Also open a bug report at the DokuWiki bug tracker about the error message being empty

**Comment by Braintelligence** [comment]
I'll try if that was the problem in a few hours. And if the writing permission is different from the one of the english lang files I will report about the missing error code for this error-constellation.

**Comment by Braintelligence** [comment]
I'm sorry I took so long.
I checked the writing permissions; they are identical for all languages folders and files. So this can't be the reason and I still can't deinstall.

I tried reinstalling (which worked) and then deinstalling again (which didn't work and produced an error div with no content, again).
I'm running the current version of the plugin on DokuWiki Hrun.

What information is relevant so we can find the reason for this behaviour?
Or alternatively:
How could I manually remove the talkpage plugin without leaving any traces or leftover code.
